client/question/components/SpeechBalloonDirectives.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directives for displaying speech balloons.
+ */
+
+tie.directive('tieSpeechBalloonContainer', ['$timeout', function($timeout) {
+  return {
+    restrict: 'E',
+    scope: {},
+    link: function(scope, element) {
+      var speechBalloonContainerElement = element[0];
+      var speechBalloonContainer =
+          angular.element(speechBalloonContainerElement);
+      speechBalloonContainer.addClass("tie-speech-balloon-container");
```

Comment:
There's some single quotes and some double quotes, use singles throughout.

Reply:
Done